css-grid-1/grid-items/grid-item-containing-block-004.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Grid item sizing</title>
+<link rel="author" title="Tomek Wytrebowicz" href="mailto:tomalecpub@gmail.com">
+<link rel="help" href="http://www.w3.org/TR/css-grid-1/#grid-items" title="4. Grid Items">
+<meta name="assert" content="A grid item is sized within the containing block defined by its grid area">
+<link rel="match" href="../../css21/reference/ref-filled-green-100px-square.xht">
+<style>
+    #grid {
+        display: grid;
+        position: absolute;
+        height: 100px;
+        width: 100px;
+        background-color: red;
+        grid-template-columns: 1fr;
+        grid-template-rows: 1fr;
+    }
+
+    #test-item{
+        background-color: green;
+        width: 100%;
+        height: 100%;
+    }
+
+</style>
+
+<body>
+    <p>Test passes if there is a filled green square and
+        <strong>no red</strong>.</p>
+    <div id="grid">
+        <div id="test-item">
```

Comment:
Main problem with this test is that it's not failing in a browser without grid support.
These tests are supposed to fail if you don't have support for grid layout.

Maybe you can simply reuse `grid-item-containing-block-003.html` and use `position: absolute;` in the grid container.